docs/getting-started/zowe-certificates-overview.md

```diff
@@ -0,0 +1,143 @@
+This is, however, not recommended for production environments.
+
+Read the following sections to learn about the key concepts of the certificates-based security in Zowe.
+
+### Digital certificates usage
+
+Digital certificates according to x.509 standard specification are the cornerstone for securing communication channels between clients and servers.
+x.509 certificates can also be used to provide identification of clients and service users.
+
+Digital certificates perform two primary functions:
+- Verification of the identity of a sender/receiver of an electronic message
+- Encryption/Decryption of the messages between the sender and the receiver.
+
+Zowe uses digital certificates as a foundational element of both, communication and identity security.
+Additionally, Zowe provides client identity validation functionality based on the ownership of the provided x.509 client certificate and the mainframe security authentication mechanism.
+
+Visit the [Zowe certificate usage](../user-guide/use-certificates.md) dedicated article, to learn details about how Zowe leverages certificates.
+
+Read the [Zowe certificate configuration overview](../user-guide/configure-certificates.md) article in the Zowe User Guide documentation to understand the various options for Zowe certificate configuration.
+
+### Public key infrastructure
+[Public Key Infrastructure (PKI)](https://en.wikipedia.org/wiki/Public_key_infrastructure) is a key aspect of internet security. PKI is both the technology and processes that make up the framework for encryption to protect and authenticate digital communications.
+PKI includes software, hardware, policies, and procedures that are used to create, distribute, manage, store, and revoke digital certificates and manage public-key encryption.
+
+Visit the following external link, to learn [How Does PKI Work](https://www.keyfactor.com/education-center/what-is-pki/#section2).
+
+Visit the following link to learn more about PKI in the context of the [z/OS Cryptographic Services](https://www.ibm.com/docs/en/zos/2.3.0?topic=planning-introducing-pki-services).
+
+### Transport Layer Security
+[Transport Layer Security (TLS)](https://en.wikipedia.org/wiki/Transport_Layer_Security) is a networking cryptography protocol that provides authentication, privacy and data integrity between two communicating computer applications.
+TLS is a successor to Secure Socket Layer (SSL), which was deprecated in 2015.
+
+***Note*** Although the transition from SSL 3.0 to TLS 1.0 happened back in 1999, people still often would refer to SSL instead of to TLS, eventually voice it tas SSL/TLS.
+
+Most importantly, TLS defines a client-server handshake mechanism to establish an encrypted and secure connection, to ensure the authenticity of the communication parties.
+During the handshake, the parties negotiate exchange algorithm, cipher suites and exchange key material to establish a stateful encrypted connection.
```

Comment:
What do you mean "negotiate"? Is there a more specific word to use in this context?

Reply:
During the handshake, the client and the server really "negotiate" what algorithm should they use for encryption, because they may have different capabilities, versions etc, so they need to agree on a common set.